tests/errors/panic.go

```diff
@@ -0,0 +1,7 @@
+package errors
+
+func PanicOnError(err error) {
```

Comment:
If you created this one, why didn't you used it in the console/login.go ?
But I would prefer to see this in a different commit or better, in a different PR.